sample-authz/src/main/java/net/trajano/ms/example/authz/SampleAuthzMS.java

```diff
@@ -1,12 +1,17 @@
 package net.trajano.ms.example.authz;
 
+import javax.annotation.PostConstruct;
 import javax.ws.rs.core.Application;
 
+import org.springframework.beans.factory.annotation.Autowired;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.springframework.beans.factory.annotation.Autowired'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)